task2/utils.py

```diff
@@ -0,0 +1,23 @@
+import logging
+import os
+
+
+def get_logger(log_path, log_name, log_level):
+    log_level_info = {'logging.DEBUG': logging.DEBUG,
+                      'logging.INFO': logging.INFO,
+                      'logging.WARNING': logging.WARNING,
+                      'logging.ERROR': logging.ERROR,
+                      }
+    set_log_level = log_level_info.get(log_level, logging.ERROR)
```

Comment:
@patrikprovaznik you can do .get above when you are creating the dict.. no need to save the variable